Jellyfin.Plugin.Bookshelf/Providers/GoogleBooks/GoogleBooksImageProvider.cs

```diff
@@ -84,9 +84,7 @@ public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, Cancell
 
             using var response = await httpClient.GetAsync(url, cancellationToken).ConfigureAwait(false);
 
-            #pragma warning disable CA2007
-            await using var stream = await response.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false);
-            #pragma warning restore CA2007
+            using var stream = await response.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
I would revert this change

Reply:
I'm wondering if maybe the `stream` assignment could be removed entirely by replacing `response.Content.ReadAsStreamAsync` with `response.Content.ReadFromJsonAsync`, to make it consistent with how it's done in the main Jellyfin repo ?

Reply:
I agree with this suggestion